src/runtime/threading_backend.cc

```diff
@@ -65,8 +87,18 @@ class ThreadGroup::Impl {
 #endif
 #endif
 #if defined(__linux__) || defined(__ANDROID__)
+    CHECK_LE(threads_.size() + exclude_worker0, sorted_order_.size());
+    if (sorted_order_.size() != threads_.size()) {
+      LOG(WARNING) << "setting affinity with subset of threads!";
```

Comment:
this warning seems can be a bit frequent, instead of this, do

Do CHECK_GE(sorted_order_.size(), threads_.size());